### PR TITLE
feat: add grid, navigation and slider comp

### PR DESCRIPTION
**Pull request by PuneetP16**
GRID
---
image

NAVIGATION
---
image

SLIDER
---
image

### CODE REVIEW

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge.

File: src/components/slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+				<h1 class="h4 nav__head">Components</h1>
+				<ul class="nav__items flex-col-align-center">
+					<li class="nav__list">
+						<a
+							class="nav__links"
+							href="/src/components/avatar/avatar.html"
+							>Avatar</a
+						>
+					</li>
+					<li class="nav__list">
```
Comment (pankaj-prog) [inline src/components/slider/slider.html:52]: work on indentation

File: src/components/slider/slider.html [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                        <h3 class="h5 filter__head">Volume</h3>
+                        <ul class="range__head range__head--icon">
+                            <li class="range__amount">50</li>
+                            <li class="range__amount">150</li>
+                            <li class="range__amount">250</li>
+                        </ul>
+                        <div class="range__slider--icon">
+                            <i class='bx bxs-volume-full'></i>
+                            <input type="range" min="500" max="2500" value="1500" class="range__slider">
+                        </div>
```
Comment (pankaj-prog) [inline src/components/slider/slider.html:208]: class names are good

**Review by pankaj-prog — APPROVED** [review]

File: src/components/slider/slider.html (hunk 1 above)
Comment (PuneetP16) [reply src/components/slider/slider.html:52]: Will look into it. Thanks for the review.

**Pull request merged**